api/tests/test_manage_asset_api.py

```diff
@@ -134,7 +134,10 @@ def test_admin_can_post_asset(self, mock_verify_id_token):
             "asset_code": "IC002",
             "serial_number": "SN002",
             "model_number": self.assetmodel.model_number,
-            "purchase_date": "2018-07-10"
+            "purchase_date": "2018-07-10",
+            "processor_type": "Intel core i3",
+            "processor_speed": 2.3,
+            "screen_size": 15
         }
```

Comment:
i think we should also have a test for invalid data. eg a screen size not present in the set choices.

Reply:
i think we should also have a test for invalid data. eg a screen size not present in the set choices.
And a test to confirm an asset is created when the specs aren't provided.